internal/lpastore/update.go

```diff
@@ -130,6 +130,15 @@ func (c *Client) SendAttorney(ctx context.Context, lpa *Lpa, attorney *actor.Att
 
 	if attorney.IsTrustCorporation {
 		body.Type = "TRUST_CORPORATION_SIGN"
+		lpaTrustCorp := lpa.Attorneys.TrustCorporation
```

Comment:
you'll need to check if it's a replacement, then use `lpa.ReplacementAttorneys.TrustCorporation` instead